winter/controller.py

```diff
@@ -71,26 +93,33 @@ def parameter(self) -> inspect.Parameter:
         return self.method.signature.parameters[self.name]
 
 
-def _register_controller(controller_class):
+def _register_controller(controller_class: type) -> None:
```

Comment:
I like the idea that `Type` is for typing, `type` is for code - `type()`

Reply:
Dome